test-requirements.txt

```diff
@@ -4,6 +4,10 @@ tox==3.7.0
 Werkzeug==0.15.5
 pytest-localserver==0.5.0
 pytest-cov==2.8.1
+google-api-python-client==1.10.0
```

Comment:
Please put those in tox.ini like the boto3 dependency

Reply:
Okay sure, I'll update this.